scripts/lintcommit.lua

```diff
@@ -0,0 +1,115 @@
+function commit_message_is_ok(commit_message)
+  local commit_split = vim.fn.split(commit_message, ":")
+
+  -- Return true if the type is vim-patch since most of the normal rules don't
+  -- apply.
+  if commit_split[1] == "vim-patch" then
+    return true
+  end
+
+  -- Return false if the number of colons detected is either 0 or more than 1.
+  if vim.tbl_count(commit_split) ~= 2 then
```

Comment:
does conventional commits disallow more than one colon? spec doesn't forbid it: https://www.conventionalcommits.org/en/v1.0.0/#specification
```suggestion
  if vim.tbl_count(commit_split) < 2 then
```

Reply:
You're right, not sure why I had assumed that. Thanks.